Modules/posixmodule.c

```diff
@@ -8210,6 +8211,16 @@ os_sched_param_impl(PyTypeObject *type, PyObject *sched_priority)
     return res;
 }
 
+static PyObject *
+os_sched_param_reduce(PyObject *self, PyObject *ignored)
```

Comment:
I think it is possible to use a `Py_UNUSED` instead of calling the name ignored :)

Reply:
Thanks for the reminder. I was going to use that macro, but did not remember the exact name, and later forgot to add it.